pkg/test/framework/featurelabels/features.go

```diff
@@ -0,0 +1,18 @@
+package featurelabels
+
+type Security struct {
+	MTLS MTLS
+}
```

Comment:
nit/premature optimization: i've never actually tried this, but as this gets deeper does it make sense to make these nested? Like
```go
type Security struct {
	MTLS struct {
             someField string
        }
}
```

Or will that break when we reference it later? Just to put everything in the same place so its no so spread out

Reply:
I did not realize that was an option!

I was thinking in the long run we'd have a simple yaml file defining all features and generate types to match.  Either way, I think the end goal is that these are nested constants, which can be easily converted to a string for serialization.